### PR TITLE
Release notes for 7.17.4

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

**Review by kares — COMMENTED** [review]
:blossom: - one suggestion about the issue fixed

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -62,6 +63,31 @@ This section summarizes the changes in the following releases:
...
+
+==== Plugins
+
+*Cef Codec - 6.2.5*
+
+* [DOC] Update link to CEF implementation guide https://github.com/logstash-plugins/logstash-codec-cef/pull/97[#97]
+
+*Dns Filter - 3.1.5*
+
+* Fixed an issue where a non-string value existing in the resolve/reverse field could cause the plugin to crash
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:84]:
```suggestion
* Fixed an issue where a non-string value existing in the resolve/reverse field could cause the plugin to crash https://github.com/logstash-plugins/logstash-filter-dns/pull/65[#65]
```

**Review by karenzone — APPROVED** [review]
Builds cleanly and LGTM

**Comment by karenzone** [comment]
@kaisecheng, thank you for your Release Manager work on 7.17.4.  Please merge the release notes the day before the release to help the docs portion of the release go smoothly.  @joepeeples is the Docs Release Manager for 7.17.4.

**Review by kares — APPROVED** [review]

**Pull request merged**